common/src/View/ViewPreferencePane.cpp

```diff
@@ -247,7 +259,21 @@ namespace TrenchBroom {
             wxString iconSizes[7] = {"25%", "50%", "100%", "150%", "200%", "250%", "300%"};
             m_textureBrowserIconSizeChoice = new wxChoice(viewBox, wxID_ANY, wxDefaultPosition, wxDefaultSize, 7, iconSizes);
             m_textureBrowserIconSizeChoice->SetToolTip("Sets the icon size in the texture browser.");
+
+
             
+            wxStaticText* fontPrefsHeader = new wxStaticText(viewBox, wxID_ANY, "Fonts");
+            fontPrefsHeader->SetFont(fontPrefsHeader->GetFont().Bold());
+
+            wxIntegerValidator<unsigned int> ValidIntP;
```

Comment:
I noticed the text rendering breaks if you go as high as 90 and I froze TB by typing in 111. It looks like it's possible to restrict the range on wxIntegerValidator with something like `ValidIntP.SetRange(2, 72)`